flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/ProgramTargetDescriptor.java

```diff
@@ -25,56 +25,28 @@
  */
 public class ProgramTargetDescriptor {
 
-	private final String clusterId;
-
 	private final String jobId;
 
-	private final String webInterfaceUrl;
-
-	public ProgramTargetDescriptor(String clusterId, String jobId, String webInterfaceUrl) {
-		this.clusterId = clusterId;
+	public ProgramTargetDescriptor(String jobId) {
 		this.jobId = jobId;
-		this.webInterfaceUrl = webInterfaceUrl;
-	}
-
-	public String getClusterId() {
-		return clusterId;
 	}
 
 	public String getJobId() {
```

Comment:
Maybe out of this patch but any reason we stick to `String` representation of `JobID`? I've checked the usage and it seems we can just use `JobID` type and print its hex string representation in `toString` method. The idea is that a concrete type is better than generic `String` representation.

Reply:
I think that's a fair point but I tried to keep the changes minimal.

Reply:
Make sense. Track as [FLINK-14958](https://issues.apache.org/jira/browse/FLINK-14958).